package.json

```diff
@@ -24,6 +24,7 @@
     "check-for-modified-files": "cd scripts && just-scripts check-for-modified-files",
     "checkchange": "beachball check  --scope \"!packages/fluentui/*\" --changehint \"Run 'yarn change' to generate a change file\"",
     "check-if-v0-v8-v9-affected": "node ./scripts/monorepo/checkIfPackagesAffected.js",
+    "check:installed-npm-package-versions": "satisfied --skip-invalid --ignore \"prettier|angular|lit|sass|@storybook/web-components|svelte|@testing-library|vue\"",
```

Comment:
using more meaningful alias. in general naming aliases by the name of the actual binary is confusing and hard to spot what on is being actually invoked